_posts/2018-09-08-terminal-tips.md

```diff
@@ -3,4 +3,27 @@ title: Terminal tips from PES Open Source
 tags: linux, bash, compilation
 ---
 
-// Start your contributions here
+*Contributed by: Aditi*		      
+## Short tricks for long commands:     
+- Use `Ctrl+A` and `Ctrl+E` to navigate to the beginning and end of the line respectively.   
+- Hit `Ctrl+R` and enter a word/phrase from a previously used super long command and it will retrieve the last used command with the term.  
+
+## Reusing previous commands
+- `!!` is handy for reusing the previous command with the current one.  
+- `!?` uses the argument of the previous command with the current one.
+
+## Bash history
+To avoid having a command added to your .bash_history file, prefix it with a space i.e. go with `  [command]` instead of `[command]`.
```

Comment:
Also while you are making the above change, markdown seems to not be adding in the space for ` [command]`
I think you'll have to use`<code>&nbsp;command</code>` instead to make it work